src/Cache/Engine/FileEngine.php

```diff
@@ -248,8 +248,8 @@ public function clear(): bool
             RecursiveIteratorIterator::SELF_FIRST
         );
         $cleared = [];
-        /** @var \SplFileInfo $fileInfo */
         foreach ($contents as $fileInfo) {
+            assert($fileInfo instanceof SplFileInfo);
```

Comment:
This is fine and won't cost much, but I wish we could just check the array type once.